index.html

```diff
@@ -6,6 +6,10 @@
 	</head>
 
 	<body>
+		<div class="darkmode">
```

Comment:
Must have a button to enable light mode

Reply:
@Nikhossain I have added  light mode button